templates/homepage.html

```diff
@@ -20,24 +20,26 @@ <h2>Getting started</h2>
         </div>
       </div>
       <hr class="is-deep">
+      <div class="u-equal-height">
```

Comment:
If this only has one child, it shouldn't have an equal height util?

Reply:
Can it just be `<div>`? Or does it need some kind of class?

Reply:
Removed